doc-content/enterprise-only/installation/nfs-server-configure-proc.adoc

```diff
@@ -0,0 +1,42 @@
+[id='nfs-server-configure-proc']
+
+= Configuring an NFS server
+
+You must deploy and configure an NFS server to provide the shared file system necessary for a {CENTRAL} clustered environment and each node client must have access to the shared file system.
+
+You must use NFS version 4.
+
+.Procedure
+
+. Configure a server to export NFS version 4 shares. For instructions about exporting NFS shares on Red Hat Enterprise Linux, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/exporting-nfs-shares_managing-file-systems[Exporting NFS shares]. For more information about creating the NFS server, see https://access.redhat.com/solutions/1355233[_How to configure NFS in RHEL 7_].
```

Comment:
It's best to just link to the TOC in books that you have no control over, then direct the user to the appropriate section in words. You never know when the author might rearrange the chapters.

Reply:
Good point, done